.github/workflows/build_status.yml

```diff
@@ -0,0 +1,27 @@
+name: Build status
```

Comment:
This was originally done because there was no way to show a test status badge for a specific build within a workflow. It looks like they've added a the ability to filter on branch, so I guess we can avoid this duplication now. I'll make the change.